File: cloud_testing/cloud_api_poc/src/main/java/cloud_api_poc/GcTesting.java
package cloud_api_poc;

import static cloud_api_poc.Config.getCredential;
import static cloud_api_poc.Constants.*;

import com.google.testing.Testing;

public abstract class GcTesting {
  private GcTesting() {}

  private static Testing testing = null;

  static {
    testing =
        new Testing.Builder(httpTransport, JSON_FACTORY, getCredential())
            .setApplicationName(applicationName)
            .build();
  }

  public static Testing get() {
    return testing;
  }
}

File: cloud_testing/kotlin_poc/src/main/kotlin/GcTesting.kt
import com.google.testing.Testing

import Config.credential
import Constants.httpTransport
import Constants.JSON_FACTORY
import Constants.applicationName

object GcTesting {

    private var testing: Testing? = null

    init {
        testing = Testing.Builder(httpTransport, JSON_FACTORY, credential)
                .setApplicationName(applicationName)
                .build()
    }

    fun get(): Testing? {
        return testing
    }
}
